problems/euler/problem71/CPlusPlus/WyattBenson/prob71.cpp

```diff
@@ -0,0 +1,94 @@
+#include <iostream>
+#include <bits/stdc++.h>
+#include <string>
+using namespace std;
+
+struct node{
+	int num;
+	int denom;
+	float val;
+
+	struct node*  next;
+
+	void setVal(){
+		val = (float)(num)/(float)(denom);
+	}
+};
+void insert(struct node** head, int n, int d){
```

Comment:
Maybe add a method header comment here explaining what the method does in some level of detail